### PR TITLE
bootstrap: Remove the logging workarounds and improve the UI

### PR DESCRIPTION
**Pull request by ldoktor**
This PR is not (yet) for merging, it demonstrates how to avoid ugly workarounds in Avocado-vt to display output and to use the logging streams prepared by Avocado. This might not work properly when running Avocado-vt from Autotest so I'm waiting for responses on mailing list and I'll forward them here.

Anyway the first commit just contains just minor adjustments like log levels and wording, the second one is the major change which removes the custom handling, starts using the `avocado.app` which, as a coincidence, removes the (in my eyes unnecessary) formatting and adds colors on supported terminals (which improves the overall experience).

It fixes the https://github.com/avocado-framework/avocado/issues/1638

### CODE REVIEW

**Comment by ldoktor** [comment]
[issue: vt-bootstrap warns about no logging handler](https://trello.com/c/wPMZ8bHg/877-issue-vt-bootstrap-warns-about-no-logging-handler)

File: virttest/bootstrap.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def verify_recommended_programs(t_type):
             found = None
             try:
                 found = utils_path.find_command(cmd)
-                logging.info('%s OK', found)
+                logging.debug('%s OK', found)
```
Comment (clebergnu) [inline virttest/bootstrap.py]: Typo on commit: "actual info**r** regarding"

File: virttest/bootstrap.py [hunk 2]
```diff
@@ -894,6 +894,4 @@ def bootstrap(options, interactive=False):
     logging.info("")
     step += 1
     logging.info("%d - If you wish, you may take a look at the online docs for "
-                 "more info", step)
-    logging.info("")
-    logging.info(online_docs_url)
+                 "more info: %s", step, online_docs_url)
```
Comment (clebergnu) [inline virttest/bootstrap.py]: This one is missing:

```diff
diff --git a/virttest/bootstrap.py b/virttest/bootstrap.py
index 74b1e5d..0f51a5e 100644
--- a/virttest/bootstrap.py
+++ b/virttest/bootstrap.py
@@ -278,8 +278,8 @@ def sync_download_dir(interactive):
             diff_result = process.run(
                 diff_cmd, ignore_status=True, verbose=False)
             if diff_result.exit_status != 0:
-                logging.info("%s result:\n %s",
-                             diff_result.command, diff_result.stdout)
+                logging.debug("%s result:\n %s",
+                              diff_result.command, diff_result.stdout)
                 answer = genio.ask('Download file "%s" differs from "%s". '
                                    'Overwrite?' % (dst_file, src_file),
                                    auto=not interactive)
```

And the diff looks bad all in blue (when the next commit is applied).

File: avocado_vt/plugins/vt_bootstrap.py [hunk 3]
```diff
@@ -103,6 +103,11 @@ def run(self, args):
         args.show_job_log = False
         args.test_lister = True
 
+        # Enable root logger as some Avocado-vt libraries use that
+        handler = logging.StreamHandler()
+        handler.setLevel(logging.DEBUG)
+        logging.getLogger("").addHandler(handler)
```
Comment (clebergnu) [inline avocado_vt/plugins/vt_bootstrap.py]: OK.

**Comment by ldoktor** [comment]
@clebergnu, I fixed the typo and rebased it...

**Comment by clebergnu** [comment]
@ldoktor nice, did you also address this: https://github.com/avocado-framework/avocado-vt/pull/905/commits/2110772aae615c26f4671ee03856f468fecc0572#r108189719 ?

**Comment by ldoktor** [comment]
@clebergnu I'm sorry I missed that comment, updated...

**Pull request merged**